All Core Devs Meetings/Meeting 97.md

```diff
@@ -0,0 +1,562 @@
+# All Core Devs Meeting 97 Notes
+
+### Meeting Date/Time: Friday, Oct 2 2020, 14:00 UTC
+
+### Meeting Duration: 1:24 hrs
+
+### [GitHub Agenda](https://github.com/ethereum/pm/issues/211)
+
+### [Audio/Video of the meeting](https://youtu.be/v5Q5WPdN1jk)
+
+### Moderator: Hudson Jameson
+
+### Notes: Alita Moore
+
+---
+
+# Summary
+
+## Decisions Made
+
+| Decision Item | Description |
+| ------------- | ----------- |
+
+
+**97.1.1**: The expected EIPs in YOLO V2 are the two EIPs from YOLO V1 plus 2929
+**97.1.2**: Tomasz Stanczak to be the champion of EIP 2935
+**97.1.3**: Nothing else should go into V3 and potentially adopting Tomasz's idea -- to be fleshed out offline
+**97.1.4**: Don't include 2565 in YOLO V3 until the benchmarks are done and clients are more assured -- reassess if done within two weeks
+**97.1.5**: V1: 2315, 2537; V2: 2929; V3: 2718, 2930 -- 2935 (no YOLOs), 2565 -- determined next meeting if going into V3
+
+## Actions Required
+
+| Action Item | Description |
+| ----------- | ----------- |
+
+
+**97.2.1**: Ansger to bring back up CFI for EIP 2938 in two weeks
+
```

Comment:
Please have either as a table or as a list. Tables are made separating columns with "|". Lists are made using "- " or "* ".

Reply:
oh that's a typo, will fix it, thanks!